### PR TITLE
add classes to the repeat loop

### PR DESCRIPTION
**Pull request by ptmkenny**
https://github.com/lhz516/react-h5-audio-player/issues/126

For simplicity, I added the general loop class to the button, and then the on/off classes to the loop icon itself.

### CODE REVIEW

**Comment by lhz516** [comment]
Thanks for the PR. I'm actually considering adding the class to the player's [wrapper](https://github.com/lhz516/react-h5-audio-player/blob/4378225b2d7e4d226539981db2f3c88f6baec2d8/src/index.tsx#L698) `div` so that we can customize anything inside the wrapper based on the loop status. For example:
```css
.rhap_loop-on .rhap_repeat-button i {
  // overwrite
}
```
Could you make the change?

**Comment by ptmkenny** [comment]
Ok, updated!

File: src/index.tsx [hunk 1]
```diff
@@ -491,7 +491,7 @@ class H5AudioPlayer extends Component<PlayerProps> {
           <button
             key={key}
             aria-label={loop ? 'Enable Loop' : 'Disable Loop'}
-            className="rhap_button-clear rhap_repeat-button"
+            className="rhap_button-clear rhap_repeat-button rhap_loop"
```
Comment (lhz516) [inline src/index.tsx:494]: There's already `rhap_repeat-button`, and we don't need an extra class here, right?
```suggestion
            className="rhap_button-clear rhap_repeat-button"
```

File: src/index.tsx [hunk 2]
```diff
@@ -686,6 +686,7 @@ class H5AudioPlayer extends Component<PlayerProps> {
       style,
     } = this.props
     const loop = this.audio.current ? this.audio.current.loop : loopProp
+    const loopClass = loop ? 'rhap_loop-on' : 'rhap_loop-off'
```
Comment (lhz516) [inline src/index.tsx:689]: Looks good to me.

File: src/index.tsx (hunk 1 above)
Comment (ptmkenny) [reply src/index.tsx]: I overlooked `rhap_repeat-button`. 😅  I've removed it.

**Pull request merged**